src/js/app.js

```diff
@@ -190,6 +190,10 @@
     window.history.replaceState({}, "", baseUrl + params);
   };
 
+  function getJiraUrl(domain) {
+    return "https://issues." + domain + ".net/browse/";
```

Comment:
Pretty cool! Maybe some domains do not end with `.net`, could you support the complete domain name?